index.html

```diff
@@ -145,7 +146,45 @@ <h4>Skills</h4>
                     <li class="inBox">Terminal</li>
                 </ul>
             </div> 
-    </section>   
+    </section>  
+    <footer>
+        <div class="space"></div>
+        <h2 class="getStar">Get started</h2>
+        <p id="headline">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+        <form class="b" 
+        action="https://formspree.io/f/xnqrlgak" method="post">
+            <div class="boxForm">
+                <label class="form-name" for="name"> 
+                    your name
+                    <input class="form-name-contend" type="text" id="name" name="user-name" maxlength="30" placeholder="enter full name" required>
+                </label>
+                <label class="form-name" for="email">
+                    your email
+                    <input class="form-name-contend" type="email" id="mail" name="user_email" placeholder="your email" required>
+                </label>
+                <label for="msg">
+                    <textarea class="form-text" id="msg" name="user_message" maxlength="500" placeholder="enter your message here..." required></textarea>
+                </label>
+                </div>
+            <div class="button">
+                <button class="blackBoton" type="submit">
+                    <a class="textBoton" href="#">Star collaboration</a>
+                </button>
+            </div>
+        </form>
+        <div class="contenfoo">
+            <button class="see resume">
+                <a class="textBoton resume">Get my resume <img class="get-my-resume-image" src="./images/Union (2).png"></a>
+            </button>
+            <ul class="rsFooter" >
+                <li class="rrss"><img src="images/redes sociales/Vector (3).png" alt="GitHUB link"> </li>
+                <li class="rrss"><img src="images/redes sociales/Vector (4).png" alt="Twitter link"></li>
+                <li class="rrss"><img src="images/redes sociales/Vector (5).png" alt="LinkedIn link"></li>
+                <li class="rrss"><img src="images/redes sociales/Vector (6).png" alt="M link"></li>
+                <li class="rrss"><img src="images/redes sociales/Vector (7).png" alt="RabitHand link"></li>
+            </ul>
+        </div>
```

Comment:
-To improve the HTML structure of your webpage, I suggest you remove this `div` with class-name `contenfoo` from the contact-form section and place it just after the section, you can also consider wrapping it in a `footer` tag. Please fix this 😄